src/operators/ai.onnx/ConvTranspose/11/prepare_operator__ai_onnx__convtranspose__11.c

```diff
@@ -0,0 +1,182 @@
+    // TRACE_FATAL(0, !op_ctx->kernel_shape, "malloc failed");
+    // op_ctx->n_output_padding = a_output_padding?a_output_padding->n_ints:default_n_output_padding;
+    // op_ctx->output_padding = a_output_padding?a_output_padding->ints:ARRAYDUP(default_output_padding,default_n_output_padding);
+    // TRACE_FATAL(0, !op_ctx->output_padding, "malloc failed");
+    // op_ctx->n_output_shape = a_output_shape?a_output_shape->n_ints:default_n_output_shape;
+    // op_ctx->output_shape = a_output_shape?a_output_shape->ints:ARRAYDUP(default_output_shape,default_n_output_shape);
+    // TRACE_FATAL(0, !op_ctx->output_shape, "malloc failed");
+    op_ctx->n_pads = a_pads?a_pads->n_ints:default_n_pads;
+    op_ctx->pads = a_pads?ARRAYDUP(a_pads->ints,op_ctx->n_pads):createArray(default_n_pads, default_pad);
+    TRACE_FATAL(0, !op_ctx->pads, "malloc failed");
+    op_ctx->n_strides = a_strides?a_strides->n_ints:default_n_strides;
+    op_ctx->strides = a_strides?ARRAYDUP(a_strides->ints,default_n_strides):createArray(default_n_strides, default_stride);
+    TRACE_FATAL(0, !op_ctx->strides, "malloc failed");
+
+    // TRACE_VAR(2, true, op_ctx->auto_pad, "\"%s\"");
+    TRACE_ARRAY(2, true, op_ctx->dilations, , op_ctx->n_dilations, "%" PRId64);
+    // TRACE_VAR(2, true, op_ctx->group, "%" PRId64);
+    // TRACE_ARRAY(2, true, op_ctx->kernel_shape, , op_ctx->n_kernel_shape, "%" PRId64);
+    // TRACE_ARRAY(2, true, op_ctx->output_padding, , op_ctx->n_output_padding, "%" PRId64);
+    // TRACE_ARRAY(2, true, op_ctx->output_shape, , op_ctx->n_output_shape, "%" PRId64);
+    TRACE_ARRAY(2, true, op_ctx->pads, , op_ctx->n_pads, "%" PRId64);
+    TRACE_ARRAY(2, true, op_ctx->strides, , op_ctx->n_strides, "%" PRId64);
+
+    /* INITIALIZE OUTPUTS DATA_TYPE AND SHAPE HERE */
+
+    const int strideX = op_ctx->strides[1];
+    const int strideY = op_ctx->strides[0];
+
+    const int dilationsX = op_ctx->dilations[1];
+    const int dilationsY = op_ctx->dilations[0];
+
+    const int padStartY = op_ctx->pads[0];
+    const int padStartX = op_ctx->pads[1];
+    const int padEndY = op_ctx->pads[2];
+    const int padEndX = op_ctx->pads[3];
+
+    int outputSizeX = calcOutputSize(inputSizeX, kernelSizeX, strideX, dilationsX, padStartX, padEndX);
+    int outputSizeY = calcOutputSize(inputSizeY, kernelSizeY, strideY, dilationsY, padStartY, padEndY);
+
+    o_Y->n_dims       = i_X->n_dims;
+    o_Y->dims         = malloc(o_Y->n_dims * sizeof(int64_t));
+    o_Y->has_raw_data = 0;
+    o_Y->data_type    = ONNX__TENSOR_PROTO__DATA_TYPE__FLOAT;
+    o_Y->n_float_data = outputSizeX * outputSizeY * outputChannels;
+
+    o_Y->dims[0] = 1;
+    o_Y->dims[1] = outputChannels;
+    o_Y->dims[2] = outputSizeY;
+    o_Y->dims[3] = outputSizeX;
+
+    /* MALLOC OUTPUT TENSORS HERE */
+
+    //mallocTensorData(o_Y);
+    o_Y->float_data = (float*)malloc(outputSizeX * outputSizeY * outputChannels * sizeof(float));
```

Comment:
`mallocTensorData` is an actual helper function :D
either use this one or add a `TRACE_FATAL`